test/e2e/specs/fakerp/command.go

```diff
@@ -38,6 +38,9 @@ var _ = Describe("Command tests [Command][Fake][LongRunning]", func() {
 		err := azurecli.OpenShiftManagedClustersAdmin.RunCommand(context.Background(), os.Getenv("RESOURCEGROUP"), os.Getenv("RESOURCEGROUP"), vm, "RestartKubelet")
```

Comment:
further down this file, you need to update the check for the success of the ConditionFunc in the wait.PollImmediate call (because we now need to check for the success of 3 commands and not 2) from:

```
return count == 2, nil
```

to 

```
return count == 3, nil
```

Reply:
Will update.